### DISCUSSION TITLE
PrimeVue v4 and Tailwind presets

### DISCUSSION THREAD

**Discussion by panruza** [primefaces/community#1925] [PrimeVue Templates] [answered]
Hello, I am trying to prepare our application for the next version of PrivmeVue4. We are using the Tailwind Lara presets (from the builder), but the presets are still for version 3. Is there any possibility to download Lara presets for 4 beta version? Thank you

**Comment by tugcekucukoglu** [comment, marked as answer]
Hello, when 4.0.0-rc.1 released, we will start working on Tailwind presets.

Thank you.

**Reply by thomaslecoeur** [reply]
Hello !

Any update on the progress of the integration of tailwind presets in V4?

Thanks for the amazing work!
